core/unix/src/TUnixSystem.cxx

```diff
@@ -40,10 +40,15 @@
 #include <algorithm>
 #include <atomic>
 
+#ifdef __linux__
+#include <syscall.h>
+#endif
+
 //#define G__OLDEXPAND
 
 #include <unistd.h>
 #include <stdlib.h>
+#include <poll.h>
```

Comment:
@bbockelm Since fSignals in TSystem has been also defined as TFdSet, should we change the data structure in the new signal handling?